cli/running/running.go

```diff
@@ -126,6 +126,14 @@ type providerImpl struct {
 	instanceCtx *InstanceCtx
 }
 
+type SkipClusterConfig int
+
+const (
+	SkipConfig       SkipClusterConfig = iota // Skip cluster config loading at all.
+	SkipConfigErrors                          // Skip errors at cluster config loading.
+	LoadConfig                                // Load cluster config, trigger an errors if not.
```

Comment:
The idiomatic way to name a enums in Go:
```suggestion
	ConfigSkip       SkipClusterConfig = iota // Skip cluster config loading at all.
	ConfigLoadWIthErrors                          // Skip errors at cluster config loading.
	ConfigLoad                                // Load cluster config, trigger an errors if not.
```

Reply:
```
LoadMode
```

```
LoadConfigSkip
LoadConfigSkipErrors
LoadConfigAlways
```